docs/sources/installation/windows.md

```diff
@@ -21,18 +21,10 @@ installation.
 
 **Important:** After you've downloaded the zip file and before extracting it, make sure to open properties for that file (right-click Properties) and check the `unblock` checkbox and `Ok`.
```

Comment:
It looks like the Configure section applies only to the standalone binaries, since you do not download a zip file for the installer. Is that correct?